test/test_exporter.py

```diff
@@ -190,55 +191,63 @@ def test_dynatrace_metadata_enrichment_with_default_tags(
-        self._update_value(aggregator, 10)
-        record = self._create_record(aggregator)
+        metric = self._create_record(self._create_sum_data_point(10))
 
         exporter = DynatraceMetricsExporter(
             default_dimensions=default_tags,
             export_dynatrace_metadata=True)
         exporter._is_delta_export = True
-        result = exporter.export([record])
+        result = exporter.export([metric])
 
-        self.assertEqual(MetricsExportResult.SUCCESS, result)
+        self.assertEqual(MetricExportResult.SUCCESS, result)
         mock_post.assert_called_once_with(
             self._ingest_endpoint,
             data="my.instr,tag1=tv1,tag2=tv2,l1=v1,l2=v2,"
                  "dt_mtag1=value1,dt_mtag2=value2,"
-                 "dt.metrics.source=opentelemetry count,delta=10\n",
+                 "dt.metrics.source=opentelemetry count,delta=10 "
+                 + str(self._test_timestamp_millis)
+                 + "\n",
             headers=self._headers)
 
-
     @patch.object(requests.Session, 'post')
     @patch('dynatrace.metric.utils.dynatrace_metrics_api_constants'
            '.DynatraceMetricsApiConstants.payload_lines_limit')
-    def test_invalid_metricname_skipped(self, mock_const, mock_post):
+    def test_fail_to_serialize_skipped(self, mock_const, mock_post):
         mock_post.return_value = self._get_session_response()
         mock_const.return_value = 2
 
-        records = []
+        metrics = []
         for n in range(4):
-            aggregator = aggregate.SumAggregator()
-            self._update_value(aggregator, n)
+            data_point = self._create_sum_data_point(n)
 
             if n == 3:
                 # for the last metric, we create one with an invalid name
-                records.append(MetricRecord(
-                    DummyMetric(""), self._labels, aggregator, Resource({})
-                ))
+                # this will only fail when calling the serializer
+                metric = Metric(attributes=self._attributes,
+                                name=".",
+                                point=data_point,
+                                description="",
+                                unit="1",
+                                resource=Resource({}),
+                                instrumentation_info=InstrumentationInfo(
+                                    name="dynatrace.opentelemetry.metrics.export",
+                                    version="0.0.1"))
+                metrics.append(metric)
             else:
-                records.append(self._create_record(aggregator))
+                metrics.append(self._create_record(data_point))
 
-        first_expected = "my.instr,l1=v1,l2=v2,dt.metrics.source=opentelemetry count,delta=0\nmy.instr,l1=v1,l2=v2,dt.metrics.source=opentelemetry count,delta=1\n"
+        first_expected = "my.instr,l1=v1,l2=v2,dt.metrics.source=opentelemetry count,delta=0 {0}\nmy.instr,l1=v1," \
```

Comment:
can you put the linebreaks next to the `\n`s, so its easier to read?

Reply:
Ah yes, auto-format issue. Fixed in [e565f52](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/22/commits/e565f52f841c132dfbc9c483a70099425cd2edb7). :slightly_smiling_face: